boards/wiznet/w5500_evb_pico2/support/openocd.cfg

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2024 TOKITA Hiroshi
```

Comment:
2025?

Reply:
Thanks for your pointing out.
Fixed it.